.semaphore/pre-release.yml

```diff
@@ -23,7 +23,7 @@ blocks:
         - name: "Publish to NPM"
           commands:
             - nvm use
-            - npx lerna bootstrap
             - yarn
-            - yarn lerna version prerelease --yes
-            - yarn lerna publish from-package --yes --dist-tag next
+            - test -f .changeset/pre.json || yarn changeset pre enter next
+            - yarn changeset version
+            - git status --porcelain && git add . && git commit -m"Pre-release" --no-verify && yarn changeset publish && git push origin master --follow-tags --no-verify
```

Comment:
why do you need to run `git status --porcelain` I thought porcelain is just a "stable" output so you could parse it, but I think you are using it to see if there is a wrong or missing yarn lock or missing readme file, before even trying the release?

Reply:
Yes, just checking if there are any changes, if there're changes we will commit and publish and push. Otherwise just stop. 
I googled the `porcelain` option, because they said it's machine parsable results. To be honest I didn't try if `-s` works or not, `--porcelain` works pretty well in my testing repo.